tests/phpunit.xml

```diff
@@ -2,6 +2,7 @@
   <testsuites>
     <testsuite name="Services Twilio Test Suite">
       <directory>./</directory>
+      <exclude>WorkflowTest.php</exclude>
```

Comment:
This is excluded because the CI job failed on PHP 5.3 since the JsonSerializable class doesn't exist on PHP 5.3; only 5.4 and up. This is sort of the main orchestrator of the marshaling / un-marshaling of json. We could support PHP 5.3, but this would involve ripping up the current implementation in favor of something a bit more complicated and manual.

https://travis-ci.org/twilio/twilio-php/jobs/98179582